backend/app/api/endpoints/internal/chat_storage.py

```
@@ -944,6 +944,137 @@ async def get_chat_history(
+        )
+    return SubtaskListResponse(session_id=session_id, subtasks=summaries)
+
+
+@router.get(
+    "/history/{session_id}/subtasks/{subtask_id}",
+    response_model=SubtaskRecordResponse,
+)
+async def read_history_subtask(
+    session_id: str,
+    subtask_id: int,
+    db: Session = Depends(get_db),
```

Ruff (0.15.21)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)